migreringer/src/main/resources/db/migration/defaultDS/4.1/V4.1_16__TFP-2836-svp-avklart-opphold.sql

```diff
@@ -0,0 +1,24 @@
+create sequence SEQ_SVP_AVKLART_OPPHOLD minvalue 1000000 increment by 50;
+
+create table SVP_AVKLART_OPPHOLD
+(
+    ID                      NUMBER(19) not null constraint PK_SVP_AVKLART_OPPHOLD primary key,
+    SVP_TILRETTELEGGING_ID  NUMBER(19) constraint FK_AVKLARRT_OPPHOLD references SVP_TILRETTELEGGING,
+    FOM                     DATE not null,
+    TOM                     DATE not null,
+    SVP_OPPHOLD_ARSAK           VARCHAR2(25 char) not  null,
+    OPPRETTET_AV            VARCHAR2(20 char) default 'VL' not null,
+    OPPRETTET_TID           TIMESTAMP(3) default systimestamp not null,
+    ENDRET_AV               VARCHAR2(20 char),
```

Comment:
Vil man kunne endre disse entitetene? Om ikke så er det kanskje greit å droppe de to siste, og bruke `BaseCreateableEntitet` istdf `BaseEntitet` i `SvpAvklartOpphold`

Reply:
Antagelig vil du det siden det er saksbehandler som setter disse i et aksjonspunkt og dermed kan endre

Reply:
Regner med at det håndteres likt som IAY ved å kopiere hele grunnlaget og deaktivere det gamle så da er det kun nyttig i `SvpGrunnlagEntitet` de andre entitetene kan godt leve uten endret tror jeg.

Reply:
Har sjekket databasen Michal og du har helt rett..alle andre tabellen enn SVP_GRUNNLAG har null  endret_av og endret_tid. Jeg bytter til BaseCreateableEntitet. Bra innspill!